src/NetscriptFunctions/Corporation.ts

```diff
@@ -730,7 +730,22 @@ export function NetscriptCorporation(
       const divisionName = helper.string("hireEmployee", "divisionName", _divisionName);
       const cityName = helper.city("hireEmployee", "cityName", _cityName);
       const office = getOffice(divisionName, cityName);
-      return office.hireRandomEmployee();
+      const employee = office.hireRandomEmployee();
+      if (employee === undefined) return undefined;
+      return {
```

Comment:
how about we do this inside the hireRandomEmployee?

Reply:
the return value of hireRandomEmployee is used on line 55 of src/Corporation/OfficeSpace.ts
also it makes more sense to me to sanitize it at the edge between internal and userspace

Reply:
Should be able to just return `{...employee}` here instead of manually building a new object. This would also keep working if in the future there is a change to what characteristics an employee has.

Reply:
probably.
In this case im just mimicing what getEmployee does elsewhere in the file